style.css

```diff
@@ -177,6 +181,93 @@
     cursor: pointer;
   }
 
+  .pop-up {
+    background-color: #c1c7d0b9;
+    padding: 10% 3%;
+    position: fixed;
+    overflow-y: scroll;
+    bottom: 0;
+    top: 0;
+  }

```

Comment:
- [ ] As the previous code reviewer suggested, Please 🙏🏽 kindly make the background transparent, you could use `opacity: 80%` just for the gray section.

> By the way, the background value of the pop-up window should be transparent, just like the Figma design

- [ ] **[OPTIONAL]** kindly correct the size of the pop-up window for the mobile version, I couldn't see the close button, for a better user experience. 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/202404211-58e6c537-5c1e-44c1-b2c0-4e56d72748ef.png)

Reply:
Hello @franclobo. Thank you for your kind review ☺☺☺ But there are a few confusions that I am having. It would be much helpful for me if you can clarify those. 👌👌👌
- I have used the opacity in the hex code of the background color and I am not finding any error in my browser.
- The mobile version close button and everything work for me in the exact same dimensions. 
I have attached the screenshot. It would be great if you see them. 👇🏽
![Screenshot of the issue](https://user-images.githubusercontent.com/64170307/202427320-07d164b2-9a43-46a1-8ba3-63203667a9a8.jpg)

_Kindly help me understand what I should fix now. I would be grateful._ 👲👲👲

Reply:
Hello, @PrangonGhose I really appreciate ❤️ that you reached out to me I'm sorry for that 😢. At the time I reviewed it looked like in the picture that I took. I reviewed it now and it looks like in your picture **it should be approved in that case**. If you need to add some changes to your repo to have a new re-review **[OPTIONAL]** kindly add the LICENCE file to your repo, if you need help you could review this 👉🏽 [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) 👈🏽 and make sure that the link on the README file works as expected otherwise **you can request a second opinion** using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

Reply:
@franclobo Thank you for the reply. ❤ I really appreciate the help. I updated the license as you mentioned and I submitted for my final review. Hope to get approved. 🤞🤞🤞